build.gradle

```diff
@@ -55,18 +55,12 @@ subprojects {
         buildTasks = ['build']
         git {
             // To release from any branch
-            requireBranch = null
+            requireBranch.set('')
         }
     }
```

Comment:
A general question: is this gradle plugin currently still used to release the official versions? If not, then this could be removed. It currently also includes a deprecation warning see also (https://github.com/researchgate/gradle-release/issues/379)